packages/atlas/src/components/Fee/Fee.stories.tsx

```diff
@@ -12,18 +13,20 @@ export default {
     className: { table: { disable: true } },
     variant: { table: { disable: true } },
     color: { table: { disable: true } },
+    amount: { table: { disable: true } },
   },
   args: {
     variant: 'h500',
-    amount: 0,
+    amountToken: 0,
     loading: false,
     hideOnMobile: false,
   },
 } as Meta<FeeProps>
 
 const Template: Story<FeeProps> = (args) => (
   <Wrapper>
-    <Fee {...args} />
+    {/* eslint-disable-next-line @typescript-eslint/no-explicit-any */}
+    <Fee {...args} amount={tokenNumberToHapiBn((args as any).amountToken)} />
```

Comment:
```suggestion
   <Fee {...args} amount={tokenNumberToHapiBn(args.amountToken)} />
```
I guess this should do the job.